leetcode/array.md

```diff
@@ -0,0 +1,28 @@
++ [Squares of a Sorted Array](#squares-of-a-sorted-array)

```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 Add files via upload 
 Mistakes: 
 Where have you found such homework(Add)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(files), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(via)? It is not from ['Added', 'Deleted'].